app/javascript/dashboard/modules/contact/components/ContactIntro.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="contact--intro">
+    <thumbnail
+      :src="contact.thumbnail"
+      size="64px"
+      :username="contact.name"
+      :status="contact.availability_status"
+    />
+
+    <div class="contact--details">
+      <h2 class="block-title contact--name">
+        {{ contact.name }}
+      </h2>
+      <h3 class="sub-block-title contact--work">
+        {{ contact.title }}
+        <i class="icon ion-minus-round" />
```

Comment:
Add `v-if` for company name. Other Ways the icon will always show irrespective of company name